server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel4.java

```diff
@@ -306,7 +299,7 @@ public PlatformLevel4(PlatformLevel parent, List<Object> level4AddedComponents)
   }
 
   @Override
-  public PlatformLevel configure() {
+  protected void configureLevel() {
```

Comment:
:new: ![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)This method has 387 lines, which is greater than the 100 lines authorized. Split it into smaller methods.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS138)